src/librustc_middle/query/mod.rs

```diff
@@ -884,6 +884,7 @@ rustc_queries! {
         query is_sized_raw(env: ty::ParamEnvAnd<'tcx, Ty<'tcx>>) -> bool {
             desc { "computing whether `{}` is `Sized`", env.value }
         }
```

Comment:
Here's the [chapter on the rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/query.html) on adding queries - try grepping for an existing query (e.g. `unused_generic_params` since I know that one isn't used in many places) and copy the changes.

Reply:
Ah you're probably right, I do want to add this with what I want to end up doing, so for now I'll hold off. I'll try replicating that query when I do so, as the other one I was replicating was scattered all about and hard to follow.